### DISCUSSION TITLE
error regarding github action failing on PRs from forks but working well on PRs from branches of that repo

### DISCUSSION THREAD

**Discussion by mihirm21** [community/community#64760] [Actions]
### Select Topic Area

Bug

### Body

https://github.com/mihirm21/ofep_final/actions/runs/5939840600/job/16107143808?pr=69

The above workflow fails, but the following workflow works here at the same time ->

https://github.com/mihirm21/ofep_final/actions/runs/5868948011/job/16105333920?pr=70

The action is: 
![image](https://github.com/community/community/assets/105881639/0bbd3bfb-cefe-4b33-8598-b9b9d403bf52)

The error in the first link is : 
![image](https://github.com/community/community/assets/105881639/4458c611-9279-430c-9544-534d39ead884)

The python file which the workflow runs is: 
![image](https://github.com/community/community/assets/105881639/c510c40a-52e6-4c59-8b78-4ed5a8950f1d)

__Question__: Why could it not access the named URL in error for the PR through an external fork but works well when PR is from a self-branch? Is this a bug or a restriction by GitHub?

**Comment by arafajs** [comment]
GitHub does not allow you to access resources from the forked repository in a pull request. This is a restriction by GitHub to prevent security issues.

**Reply by mihirm21** [reply]
I guess you misunderstood the question. I wanted to ask that whether could I access resources of a pull request posted on my repo through a fork, using github actions.

**Reply by arafajs** [reply]
Sorry for the misunderstanding. I think you can use the `github.event.pull_request` context in your GitHub Actions workflow to do that.

**Reply by mihirm21** [reply]
Thanks, but it doesn't work. Gives the same error. Actually, I am doing the same things through the Python script.
